### ISSUE TITLE
[Bug] pmd error

### ISSUE THREAD

**Issue by r0bb3n** [r0bb3n/sonar-quality-gate-maven-plugin#159]
**Describe the bug**
build warning

```
[INFO] --- maven-pmd-plugin:3.15.0:pmd (pmd) @ sonar-quality-gate-maven-plugin ---
[INFO] PMD version: 6.38.0
Warning:  Could not find class org.r0bb3n.maven.Util, due to: java.lang.IncompatibleClassChangeError: class net.sourceforge.pmd.lang.java.typeresolution.visitors.PMDASMVisitor has interface org.objectweb.asm.ClassVisitor as super class
```

**Comment by r0bb3n** [comment]
This is a known bug, see: https://issueexplorer.com/issue/pmd/pmd/3502
Will be fixed with maven-pmd-plugin:3.16.0.

**Comment by r0bb3n** [comment]
rolling back to version 3.14.0 and configure dependabot to skip 3.15.0. By that the fixed version 3.16.0 will be included by dependabot once it is released.
